content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -271,6 +271,12 @@ When clients connect to the VIP, their traffic is automatically transported to a
 appropriate endpoint. The environment variables and DNS for Services are actually
 populated in terms of the Service's virtual IP address (and port).
 
```

Comment:
```suggestion

### IP address allocation tracking

```
Maybe use [tabs](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#tabs) to describe and compare the two options (not sure what we call them, but basically the old way with a global allocation map in etcd that end users can't see, vs. the new alpha way).

We usually put the feature state shortcode under a heading that summarises the thing. _Not_ the change, but the thing for which the behavior is being changed.

Reply:
there is no much visible change except the new objects
```
 kubectl get services
NAME         TYPE        CLUSTER-IP        EXTERNAL-IP   PORT(S)   AGE
kubernetes   ClusterIP   2001:db8:1:2::1   <none>        443/TCP   3d1h
aojea@aojea:~/src/kind$ kubectl get ipaddresses
NAME              PARENTREF
2001:db8:1:2::1   services/default/kubernetes
2001:db8:1:2::a   services/kube-system/kube-dns
```

do you want me to add it?